.travis.yml

```diff
@@ -1,9 +1,12 @@
 language: node_js
+sudo: required
+services:
+  - docker
 node_js:
   - "0.10"
   - "4.4.3"
-sudo: false
-before_install:
-  - npm install -g npm@2.13.5
-  - npm install -g grunt-cli
+before_install: 
+  - git clone https://gist.github.com/psturc/8f386fd74d8b34faf113ff14d7638529
+  - cp 8f386fd74d8b34faf113ff14d7638529/build.sh .
```

Comment:
Perhaps, we might think about moving this to a CI/test/automation etc repository ?